### PR TITLE
Removing refresh all 

### PR DESCRIPTION
**Pull request by denis-yuen**
Finishing removal of refresh all from the UI

### CODE REVIEW

**Comment by denis-yuen** [comment]
FYI, differential loading died three times in a row with the message shown in Slack

File: package.json [hunk 1]
```diff
@@ -64,8 +64,8 @@
     "zone.js": "~0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.803.25",
```
Comment (coverbeck) [inline package.json]: Comment for line 6 -- if you update to alpha.4, the compiler should catch any usages of the endpoints you have removed (although you probably already caught them all)

File: package.json [hunk 2]
```diff
@@ -64,8 +64,8 @@
...
-    "@angular/cli": "^8.3.25",
+    "@angular-devkit/build-angular": "^0.803.21",
```
[the remark this answers is not in the document]
Comment (denis-yuen) [reply package.json]: yup

File: package.json (hunk 1 above)
Comment (denis-yuen) [reply package.json]: Worth a shot

**Comment by denis-yuen** [comment]
`ci/circleci: integration_test_3` died three times with 
Screenshot from 2020-03-11 18-14-32

**Comment by coverbeck** [comment]
> `ci/circleci: integration_test_3` died three times with

Probably my fault for suggesting you upgrade to the latest web service :( Although it probably represents a real issue.

**Comment by denis-yuen** [comment]
Note for self, there is a 500 error visible in the cypress video for `/api/workflows/published`

**Comment by garyluu** [comment]
I looked into this a bit, updating the webservice to the current develop will automatically fix the test failure.

**Pull request merged**